doc/glossary.rst

```diff
@@ -195,6 +195,10 @@
                 The CRUSH data placement rule that applies to a particular
                 pool(s).
 
+        DAS
```

Comment:
I usually see `Direct` not `Directly` but close enough

Reply:
No, you're right. You're right.